addons/sourcemod/scripting/gokz-errorboxfixer.sp

```diff
@@ -0,0 +1,66 @@
+#include <sourcemod>
+#include <sdktools>
+
+#include <gokz>
```

Comment:
For updater URL and version only. Could this be moved to not required maybe? Not that I see why you would want this plugin without running a kz server? 🤔

Reply:
The main include file does not add dependency to any GOKZ plugins, whether you have it under optional or not it does not matter.